codecov_cli/services/staticanalysis/analyzers/python/node_wrappers.py

```diff
@@ -10,6 +10,43 @@ def visit(self, node):
         for c in node.children:
             self.visit(c)
 
+    def _is_function_docstring(self, node):
+        """Skips docstrings for funtions, such as this one.
+        Pytest doesn't include them in the report, so I don't think we should either,
```

Comment:
Is this specific to pytest or just coveragepy in general?

Reply:
coveragepy in general